lib/approveAndMerge.js

```diff
@@ -74,4 +82,27 @@ async function approveAndMerge(
   )
 }
 
+function getPackageDetails(pullRequest) {
+  // dependabot branch names are in format "dependabot/npm_and_yarn/pkg-0.0.1"
+  // or "dependabot/github_actions/fastify/github-action-merge-dependabot-2.6.0"
+  const nameAndVersion = pullRequest.head.ref.split('/').pop().split('-')
+  const version = nameAndVersion.pop() // remove the version
+  return {
+    name: nameAndVersion.join('-'),
+    version
+  }
+}
+
+function isMajorRelease(pullRequest) {
+  const expression = /bump \S+ from (\S+) to (\S+)/i
+  const match = expression.exec(pullRequest.title)
+  if (match) {
+    const [, oldVersion, newVersion] = match
+    if (oldVersion.split('.')[0] !== newVersion.split('.')[0]) {
+      return true
+    }
+  }
+  return false
+}
```

Comment:
a more robust way to do this would be to use the semver package, but I appreciate this is a solution specific to a certain package so we are probably good with this too